### PR TITLE
Linux-Adding--s-in-question

### PR DESCRIPTION
**Pull request by abdullahtahir147**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/user-and-file-management/user-management-utilities/random-password-generator.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/linux/user-and-file-management/user-management-utilities/random-password-generator.md?ref=91e53c803a350de823460e6cc21440a0683803ae)

File: linux/user-and-file-management/user-management-utilities/random-password-generator.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ i^^iAa5o^oc["+43[G]{62&S!&Mt:L]ny7qbR~g0
 
 ## Practice
 
-Generate a random 20 chararacters password using `pwgen` :
+Generate a random, secure 20 characters password using `pwgen` :
```
Comment (kapnobatai137) [inline linux/user-and-file-management/user-management-utilities/random-password-generator.md:70]: what do you think about this?

```suggestion
Using `pwgen`, generate a **secure** and **random** password with 20 characters:
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,11 +48,18 @@ Types of change:
 
 ### Fixed
 
+
+
+
+
```
Comment (kapnobatai137) [inline CHANGELOG.md:54]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -48,11 +48,18 @@ Types of change:
...
 ## August 18th 2021
 
 ### Fixed
 - [Data Analysis - Pipeline Analysis - Remove the type-in-the-gap format from both the PQ and the RQ](https://github.com/enkidevs/curriculum/pull/2847)
 
+- [Linux - Random Password Generator - Add "secure" in question](https://github.com/enkidevs/curriculum/pull/2849)
+
+
```
Comment (kapnobatai137) [inline CHANGELOG.md:62]:
```suggestion
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**